source/globalCommands.py

```diff
@@ -3670,6 +3670,14 @@ def script_reportLinkDestination(
 		ti.expand(textInfos.UNIT_CHARACTER)
 		obj: NVDAObject = ti.NVDAObjectAtStart
 		presses = scriptHandler.getLastScriptRepeatCount()
+		if (
+			obj.role == controlTypes.role.Role.GRAPHIC  # If it's a graphic, and
+			and (
+				obj.parent
+				and obj.parent.role == controlTypes.role.Role.LINK
+			)
+		):

```

Comment:
What do you think of this comment?
It explains the problem we are trying to solve more broadly.

```suggestion
		):
				# In Firefox, graphics with a parent link also expose the parents link href value.
				# In Chromium, the link href value must be fetched from the parent object. (#14779)
```